src/pydata_sphinx_theme/assets/styles/abstracts/_links.scss

```diff
@@ -137,10 +135,8 @@ $link-hover-decoration-thickness: unquote("max(3px, .1875rem, .12em)") !default;
   font-weight: 600;
   color: var(--pst-color-primary);
   @if $link-hover-decoration-thickness {
-    box-shadow: inset
-      $link-hover-decoration-thickness
-      0px
-      0px
+    border-left: $link-hover-decoration-thickness
```

Comment:
I _think_ this caused a small glitch, because the border has an actual width compared to the box-shadow?

Comparing the readthedocs preview of this PR vs the latest docs:

https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/layout.html#templates-and-components:

![image](https://github.com/pydata/pydata-sphinx-theme/assets/1020496/b01a872b-630b-4e0b-853e-a751427fb23e)

And on the PR (https://pydata-sphinx-theme--1549.org.readthedocs.build/en/1549/user_guide/layout.html#templates-and-components):

![image](https://github.com/pydata/pydata-sphinx-theme/assets/1020496/aea0038d-6b12-40f3-9c14-2715d136f25b)

Reply:
ooh, good catch @jorisvandenbossche

Reply:
Yes, thanks for catching that! Follow up PR: https://github.com/pydata/pydata-sphinx-theme/pull/1561